src/OpenTelemetry/Logs/OpenTelemetryLoggerOptions.cs

```diff
@@ -17,14 +17,28 @@
 #if NET461 || NETSTANDARD2_0
 using System;
 using System.Collections.Generic;
-using Microsoft.Extensions.Logging;
 
 namespace OpenTelemetry.Logs
 {
     public class OpenTelemetryLoggerOptions
     {
         internal readonly List<BaseProcessor<LogRecord>> Processors = new List<BaseProcessor<LogRecord>>();
 
+        /// <summary>
+        /// Gets or sets a value indicating whether or not log scopes should be included on generated <see cref="LogRecord"/>s. Default value: False.
+        /// </summary>
+        public bool IncludeScopes { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not log message should be included on generated <see cref="LogRecord"/>s. Default value: False.
+        /// </summary>
+        public bool IncludeMessage { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not log state should be parsed into <see cref="LogRecord.StateValues"/> on generated <see cref="LogRecord"/>s. Default value: False.
```

Comment:
Also add that turning this on will cause LogRecord.State be not populated and will be left null.

Reply:
This will be on my follow-up PR.